### PR TITLE
Merge develop into master, with Rob's sort-zipcode

### PR DESCRIPTION
**Pull request by johnhawkinson**
This is mostly a test to see if this is automatically assigned to Zach or how the reviewing process works.

This is a merge of what's in develop now, but via merge commits, because they made the merge work so much more easily.
Not fully tested by me so may not be ready to go. 
cc @harcod @tkjones117

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -7843,6 +7851,14 @@
...
             }
         },
+<<<<<<< HEAD
+=======
+        "node_modules/geolib": {
+            "version": "3.3.1",
+            "resolved": "https://registry.npmjs.org/geolib/-/geolib-3.3.1.tgz",
+            "integrity": "sha512-sfahBXFcgELdpumDZV5b3KWiINkZxC5myAkLk067UUcTmTXaiE9SWmxMEHztn/Eus4JX6kesHxaIuZlniYgUtg=="
+        },
+>>>>>>> develop-m
```
Comment (harcod) [inline package-lock.json:7861]: Looks like this conflict wasn't resolved.  There are 2 more of these as well.

File: package-lock.json [hunk 2]
```diff
@@ -17206,6 +17222,18 @@
...
+        "node_modules/universal-cookie": {
+            "version": "4.0.4",
+            "resolved": "https://registry.npmjs.org/universal-cookie/-/universal-cookie-4.0.4.tgz",
+            "integrity": "sha512-lbRVHoOMtItjWbM7TwDLdl8wug7izB0tq3/YVKhT/ahB4VDvWMyvnADfnJI8y6fSvsjh51Ix7lTGC6Tn4rMPhw==",
+            "dependencies": {
+                "@types/cookie": "^0.3.3",
+                "cookie": "^0.4.0"
+            }
+        },
+>>>>>>> develop-m
```
Comment (harcod) [inline package-lock.json:17236]: Conflict?

File: package-lock.json [hunk 3]
```diff
@@ -17350,6 +17378,17 @@
...
+=======
+        "node_modules/us-zips": {
+            "version": "4.0.2",
+            "resolved": "https://registry.npmjs.org/us-zips/-/us-zips-4.0.2.tgz",
+            "integrity": "sha512-ITciRwQnRUkngVennzsZdMmREcsIRAXavm0Uh2vpTYeQGbrtmZstk0McOMDxHo1n4HteXtRkmSWAqhkW2RswiQ==",
+            "engines": {
+                "node": ">=6"
+            }
+        },
+>>>>>>> develop-m
```
Comment (harcod) [inline package-lock.json:17391]: Conflict?

**Review by harcod — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (johnhawkinson) [reply package-lock.json]: Geeeeze. I don't understand how that could have happened, but `grep -lri '<<<<<<<'` says package-lock.json was the only in-tree file with that problem. Fixed and force-pushed.

**Comment by johnhawkinson** [comment]
Blocking this is the `App.js.test` failure:

```PASS  src/App.test.js (5.8 s)
  ● Console    console.error
      network error       97 |             })
       98 |             .catch((ex) => {
    >  99 |                 console.error(ex.message);
          |                         ^
      100 |                 setErrorMessage(
      101 |                     "Something went wrong, please try again later."
      102 |                 );      at src/App.js:99:25```

**Comment by johnhawkinson** [comment]
> Blocking this is the `App.js.test` failure:

Err, so.

Oh, I see. `App.test.js` is mocking/simulating a network error and Linus's code (from fda595e84c144f70e67bc290b5ad7bdc89163a55 , pr #13 i guess) is catching that and logging it.

So, I am having difficulty wrapping my head around the interaction between this local diagnostic code in App.js and the error test, so I do not know whether to resolve it by
1. removing the logging from App.js (as Rob suggests),
2. removing the forced network error test from App.test.js,
3. Adjusting the test to be more clear
4. Doing nothing
5. something else.

I am going to eat lunch and run to the post office instead. Guidance appreciated.

File: package-lock.json [hunk 4]
```diff
@@ -17206,6 +17219,16 @@
...
+        "node_modules/universal-cookie": {
+            "version": "4.0.4",
+            "resolved": "https://registry.npmjs.org/universal-cookie/-/universal-cookie-4.0.4.tgz",
+            "integrity": "sha512-lbRVHoOMtItjWbM7TwDLdl8wug7izB0tq3/YVKhT/ahB4VDvWMyvnADfnJI8y6fSvsjh51Ix7lTGC6Tn4rMPhw==",
+            "dependencies": {
+                "@types/cookie": "^0.3.3",
+                "cookie": "^0.4.0"
+            }
+        },
+>>>>>>> develop-m
```
Comment (iann) [inline package-lock.json:17231]: This is causing issues.

**Review by iann — CHANGES_REQUESTED** [review]

**Review by iann — APPROVED** [review]
It all looks pretty good to me. Performance a little laggy when you uncheck/check the Has Available Appointments box.

**Review by harcod — APPROVED** [review]

**Review by tkjones117 — APPROVED** [review]
OK - I did some clicking around and trying out zip codes, and this looks good to me!

I don't *love* the fact that you can scroll and the filter option follows you, but that shouldn't block this from getting in. 

Nice job team!

**Pull request merged**